### ISSUE TITLE
Screen Blanked

### ISSUE THREAD

**Issue by CasMat9** [mtgred/netrunner#3851]
![jinteki black screen](https://user-images.githubusercontent.com/4683711/44489340-3f429c00-a629-11e8-8ba1-cc547bc25d46.png)

My screen became entirely black other than the top row of buttons (concede, leave game, etc.). Only the play window was blanked, the other windows (chat, deck builder, etc.) were normal.

Screenshot attached,

**Comment by Layanasan** [comment]
I've had a screen blanked problem as well.

My opponent had experienced some connection issues earlier but could eventually get back into the game. He also told me that there were several times when he tried to rejoin the game and he just got a black screen.

Then, a few rounds later, I suddenly had a black screen instead of the playing surface. Tried to log in from another browser and rejoin. Tried to refresh my main window, left the game, tried to join again but it would not work, because by then my opponent played themselves. 

I've included a screen grab for demonstration purposes: 
![image](https://user-images.githubusercontent.com/18597570/52663813-bdf90180-2f07-11e9-8dd5-506886dd1366.png)

I've also saved our conversation log but as far as I can tell the blanking always happened on the other persons turn...
